scipy/stats/_continuous_distns.py

```diff
@@ -8393,6 +8393,13 @@ class gennorm_gen(rv_continuous):
     .. [1] "Generalized normal distribution, Version 1",
            https://en.wikipedia.org/wiki/Generalized_normal_distribution#Version_1
 
+    .. [2] Nardon, Martina, and Paolo Pianca. "Simulation techniques for
+           generalized Gaussian densities." Journal of Statistical
+           Computation and Simulation 79.11 (2009): 1317-1329
+
+    .. [3] "Simulate data from a generalized Gaussian distribution",
```

Comment:
This citation could be more complete; e.g., name of author (Rick Wicklin) and it was in The DO Loop blog. Not too big a deal since the address is right there, but if the URL were to change....

Reply:
Done.